### PR TITLE
CY-3499 - Add Getting Started section to UI documentation

### PR DESCRIPTION
**Pull request by qooban**
* Added basic information about tours and other ways for getting help in Cloudify Console
* Added basic information about navigation in Cloudify Console

### CODE REVIEW

File: content/working_with/console/getting-started/navigation.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+
+### Right side
+
+The following elements are presented in the right side of the top header bar:
+
+* **System status** - on hovering the status you can see more detailed information about system services
+* **Tenant menu** - click on the current tenant name to open dropdown selection menu and change the current tenant  
+* **Help menu** - click on the ![Help icon]( /images/ui/icons/help-icon.png ) to open menu with links to support pages 
+* **User menu** - click on the username to open [Edit Mode]({{< relref "working_with/console/customization/edit-mode.md" >}}), [Template Management]({{< relref "working_with/console/customization/templates-mgmt.md" >}}), [License Management]({{< relref "install_maintain/installation/manager-license.md" >}}), change password or log out
```
Comment (kubama) [inline content/working_with/console/getting-started/navigation.md:41]:
```suggestion
* **User menu** - click on the username to enter [Edit Mode]({{< relref "working_with/console/customization/edit-mode.md" >}}), open [Template Management]({{< relref "working_with/console/customization/templates-mgmt.md" >}}) or [License Management]({{< relref "install_maintain/installation/manager-license.md" >}}), change password or log out
```

File: content/working_with/console/getting-started/navigation.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+{{% note %}}
+Set of the available options can differ between admin and non-admin users. The list presents all the options available for admins.
```
Comment (kubama) [inline content/working_with/console/getting-started/navigation.md:44]:
```suggestion
Set of the available options can differ between admin and non-admin users. The list above presents all the options available for admins.
```

File: content/working_with/console/getting-started/navigation.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+{{% /note %}}
+
+
+## Left sidebar
+
+In the left side of the screen the **Pages list** is presented. Click on the name of the page to change the view.
+
+{{% note %}}
+Set of the available options can differ between admin and non-admin users. The list presents all the pages available for admins.
```
Comment (kubama) [inline content/working_with/console/getting-started/navigation.md:53]: What list?

File: content/working_with/console/getting-started/navigation.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+{{% /note %}}
+
+Detailed description about pages content can be found at [Pages]({{< relref "working_with/console/pages/_index.md" >}}) page.
```
Comment (kubama) [inline content/working_with/console/getting-started/navigation.md:56]:
```suggestion
Detailed description of pages content can be found at [Pages]({{< relref "working_with/console/pages/_index.md" >}}) page.
```

File: content/working_with/console/getting-started/starting.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
+---
+title: Starting
+category: Cloudify Console
+description: Learn how to log in and active the product.
```
Comment (kubama) [inline content/working_with/console/getting-started/starting.md:4]:
```suggestion
description: Learn how to log in and activate the product.
```

File: content/working_with/console/getting-started/navigation.md [hunk 6]
```diff
@@ -50,8 +50,7 @@ Set of the available options can differ between admin and non-admin users. The l
 In the left side of the screen the **Pages list** is presented. Click on the name of the page to change the view.
 
 {{% note %}}
-Set of the available options can differ between admin and non-admin users. The list presents all the pages available for admins.
+Set of the available options can differ between admin and non-admin users.
```
Comment (kubama) [inline content/working_with/console/getting-started/navigation.md:53]:
```suggestion
List of available pages can differ between admin and non-admin users.
```

**Review by kubama — APPROVED** [review]

**Review by ofercloudify — APPROVED** [review]

**Pull request merged**